src/absil/ilwrite.fs

```diff
@@ -2463,7 +2463,7 @@ let GenReturnAsParamRow (returnv : ILReturn) =
            StringE 0 |]  
 
 let GenReturnPass3 cenv (returnv: ILReturn) = 
-    if Option.isSome returnv.Marshal || not (isNil returnv.CustomAttrs.AsList) then
+    if Option.isSome returnv.Marshal || not (Array.isEmpty returnv.CustomAttrs.AsArray) then
         let pidx = AddUnsharedRow cenv TableNames.Param (GenReturnAsParamRow returnv)
```

Comment:
Do not convert to array just to check if it is empty. There must be a better way

Reply:
this is not a conversion the .AsList was a conversion